package.json

```diff
@@ -73,29 +75,10 @@
     "web-ledger-client": "^3.2.0"
   },
   "devDependencies": {
-    "eslint": "^5.12.1",
+    "eslint": "^6.0.0",
     "eslint-config-digitalbazaar": "^2.0.0"
   },
-  "bedrock": {
-    "browserDependencies": [
-      "@fortawesome/fontawesome-free",
-      "bedrock-quasar",
-      "bedrock-vue",
-      "vue",
-      "vue-router"
-    ],
-    "manifest": {
-      "@fortawesome/fontawesome-free": {
-        "main": null,
-        "less": [
-          "less/fontawesome.less",
-          "less/regular.less",
-          "less/solid.less"
-        ]
-      }
-    }
-  },
   "engines": {
-    "node": ">=8.9.0"
+    "node": ">=10.12.0"
```

Comment:
We're building and releasing the testnet with node 12, do we want to bump up to that as a minimum version?